### ISSUE TITLE
🐞 [Bug]: Upgrade action issue

### ISSUE THREAD

**Issue by Saurabh7019** [pnp/vscode-viva#60]
## 📝 Error caused by newline characters in overrides JSON while upgrading SPFx solution using vscode-viva upgrade action

> I am using vscode-viva to upgrade an SPFx solution. While updating the .eslintrc.js file as suggested in the upgrade report, I noticed an issue where newline characters ("\n") are being displayed as line breaks in the overrides JSON object. This is corrupting the file and causing the following error during the solution build.

    Cannot read config file: C:\RnD\helloWorld\.eslintrc.js
    Error: Invalid or unexpected token

## 👣 Steps To Reproduce

1. Open Viva Connection Toolkit extension in a SPFX project (v.1.15.0)
2. Click on the 'Upgrade' action. 
3. Follow the instruction in the spfx.upgrade.md report. 
4. Notice the line breaks in .eslintrc.js
![image](https://github.com/pnp/vscode-viva/assets/18114579/29c132a8-0204-42a9-b619-14e21934f7ab)

## 📜 Expected behavior

The value should be displayed without newline characters ("\n") interpreted as line breaks.
![image](https://github.com/pnp/vscode-viva/assets/18114579/9f88540e-ceb3-4457-a234-30a791238a9b)

## 💻 Desktop (please complete the following information):

 - OS: Microsoft Windows 10.0.22000

## 🤔 Additional context

> I am certain that it is a cli-microsoft365 issue, but I was unable to reproduce it with CLI v6.9.0. Therefore, logging the issue here.

**Comment by Adam-it** [comment]
@Saurabh7019 thanks for reporting the issue and sorry for the late reply. I will look into it ASAP 👍

**Comment by Adam-it** [comment]
yep so I managed to confirm this is CLI for Microsoft 365 related.
How to repro this:
1. download a spfx webpart sample from the pnp gallery that has exact spfx 1.15.0 version. I used the react roomchat
https://github.com/pnp/sp-dev-fx-webparts/blob/main/samples/react-roomchat/.yo-rc.json
2. using CLI run `m365 spfx project upgrade --output md > "upgrade-report.md"`
3. when checking the .md report without any md preview just plain text we will see the linebreaks were removed 

![image](https://github.com/pnp/vscode-viva/assets/58668583/cb7d935b-a4e6-4d73-be06-f082bccfa1c0)


@Saurabh7019 thank you for reporting the issue but unfortunately fixing this is out of scope of this tool.
I will open this as a new bug in the CLI repo and start checking it with other maintainers and then I will close this one.
